### ISSUE TITLE
Install oh-my-posh for all users [Windows]

### ISSUE THREAD

**Issue by xmha97** [JanDeDobbeleer/oh-my-posh#1514]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What would you like to see changed/added?

Hi, I want to install oh-my-posh for all users. (per machine)

✔️ Expected Behavior
```powershell
C:\Program Files\oh-my-posh
```
❌ Actual Behavior
```powershell
%LocalAppData%\Programs\oh-my-posh
```

**Comment by JanDeDobbeleer** [comment]
@xmha97 the issue seems to be missing quite a bit of information. How did you install it and what do you expect to happen in that case?

Inno allows to prompt the user to install to where they want BUT that probably ruins a silent install.

Whatever route we take, you can always add the executable to that path yourself as they're available in the releases tab.

**Comment by xmha97** [comment]
That is what I expect to happen in that case:

![image](https://user-images.githubusercontent.com/23485114/147711949-50c8721c-ecce-4064-978c-c37e86b34ddb.png)

![image](https://user-images.githubusercontent.com/23485114/147711674-a8d7454a-275f-454b-b5e9-563133822acd.png)

Also, I don't think silent install is ruined because the installer of many apps is in this way, and when I install them from [Chocolatey](https://chocolatey.org/), no problem occurs.

I have 3 users on my computer and since the Chocolatey packages are per machine (not any user like [Scoop](https://scoop.sh/)), all three apps are not updated.

Also, I think it's not logical to install an app three times.

_(I hope you have noticed and am sorry for the bad English)_

**Comment by JanDeDobbeleer** [comment]
@xmha97 seems possible. I'll see what I can do 👍🏻

**Comment by JanDeDobbeleer** [comment]
Available in 6.42.1

**Comment by xmha97** [comment]
> @xmha97 seems possible. I'll see what I can do 👍🏻

Thanks.
